binder/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: dask-tutorial
 channels:
   - conda-forge
+  - anaconda
 dependencies:
   - python=3.7
   - numpy=1.16.4
-  - nomkl
+  - anaconda::nomkl
```

Comment:
Is this available for all platforms? I don't think it's a package on Windows.

Reply:
Yeah I don't think we need it.

Reply:
Ya it is! Works on Windows too.

Reply:
I think we want `nomkl` to speed up the CI builds.

Reply:
If it gets conda-forge's `numpy` and `scipy`, then it will get `openblas`, which is all `nomkl` does for Linux.

Reply:
So is it fair to remove `nomkl` then and just use conda-forge?

Reply:
Agreed. Just pushed 1a558fa fixing this.